CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+cmake_minimum_required(VERSION 3.18)
+
+project( 
+    inchi 
+    VERSION 1.07
```

Comment:
We have to think about how to parametrize the version. Preferably we would have a single source of truth for the version. But maybe outside of this PR's scope.

Reply:
I agree on the single source for the version string; I guess it is possible to use several approaches, depending on how the release is handled: it can be read from a file, an env variable, from a workflow context variable, etc.

I would just avoid putting it in the sources, it's harder to extract for other tools and usually harder to find.